frontend/src/test/scala/bloop/io/ClasspathHasherSpec.scala

```diff
@@ -22,11 +22,15 @@ object ClasspathHasherSpec extends bloop.testing.BaseSuite {
     val cancelPromise = Promise[Unit]()
     val cancelPromise2 = Promise[Unit]()
     val tracer = BraveTracer("cancels-correctly-test")
-    val jars = {
-      DependencyResolution.resolve("org.apache.spark", "spark-core_2.11", "2.4.4", logger) ++
-        DependencyResolution.resolve("org.apache.hadoop", "hadoop-main", "3.2.1", logger) ++
-        DependencyResolution.resolve("io.monix", "monix_2.12", "3.0.0", logger)
-    }
+    val jars =
```

Comment:
The goal of the previous code was to force independent resolutions for every artifact so that the resolution of shared libraries is not shared. Is there a motivation to change that?

Reply:
I can change it back in this file. The `resolve()` function was modified to take a list of artefacts such that we can resolve platform libraries in one go.

Reply:
The changes in the resolve function are good, but this use site of that should not resolve everything together, so let’s pass a list of one element per every invocation.